shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -168,8 +168,8 @@ - (BOOL)isAccessibilityElement {
   //  We enforce in the framework that no other useful semantics are merged with these nodes.
   if ([self node].HasFlag(blink::SemanticsFlags::kScopesRoute))
     return false;
-  return [self node].flags != 0 || ![self node].label.empty() || ![self node].value.empty() ||
-         ![self node].hint.empty() ||
+  return ([self node].flags != 0 && [self node].flags != static_cast<int32_t>(blink::SemanticsFlags::kIsHidden)) ||
```

Comment:
Isn't .flags a bitfield?  If so, shouldn't you be checking ![self node].HasFlag(blink::SemanticsFlags::kIsHidden) instead of checking for inequality?

Reply:
I want to check if it *only* has the hidden flag. If it has no flags or if it only has the hidden flag the SemanticsObject is not a (a11y focusable) accessibilityElement.

Reply:
OK.